source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -1,10 +1,10 @@
 import AdaptiveCards_bridge
 import AppKit
 
-class ACRDateField: NSView {
+class ACRDateField: NSView, InputHandlingViewProtocol {
     private lazy var dateFormatter: DateFormatter = {
         let formatter = DateFormatter()
-        formatter.dateFormat = isTimeMode ? "HH:mm:ss" : "yyyy/MM/dd"
+        formatter.dateFormat = isTimeMode ? "HH:mm:ss" : "yyyy-MM-dd"
```

Comment:
Isn't `yyyy/MM/dd` the accepted common format by Microsoft?

Reply:
yyyy-MM-dd and yyyy/MM/dd is almost same, separators doesn't make difference. What I mean is, if the Json come as `value: 2000/03/03` or `value: 2000-03-03` it will get the same value.
We did this because the payload coming after submit button clicked had two backslashes, something like`2000\/03\/03`. But by changing the separator we got the payload as `2000-03-03`.